### PR TITLE
Update README for easy intro

### PR DESCRIPTION
**Pull request by alexisylchan**
Added some screenshots to the README.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -21,8 +11,57 @@ Developers:
 
 ## I want to use this app for my data set.
 
-Great! Let's get started.
+Great! Let's get started. (Already know Git? Head over to the Development section below.)
+
+### Get your own copy of finda
+![Fork via Github](https://raw.github.com/alexisylchan/finda/gh-pages/readme_img/fork.png)
```
Comment (paulswartz) [inline readme.md]: These should just be relative URLs:

```
![Fork via Github](readme_img/fork.png)
```

Then they'll work in anyone's repository.

File: readme.md [hunk 2]
```diff
@@ -21,8 +11,57 @@ Developers:
...
+If you already have a hosting service, you can use their upload manager or an FTP client to upload the files to your hosting service.
+
+If you do not have a hosting service, find and subscribe to one. [FatCow](http://www.fatcow.com/) is pretty okay. [Please][1] [don't][2] [use][3] [GoDaddy][4].
+
+[1]: http://breakupwithgodaddy.com/
+[2]: http://www.youtube.com/watch?v=_TbjSswtuNA#t=85
+[3]: http://msmagazine.com/blog/2013/02/04/top-five-sexist-super-bowl-ads-2013/
+[4]: http://www.missrepresentation.org/media/notbuyingit-godaddy-disappoints-again/
+
+Once you have a service, upload all of the files to the root folder (sometimes referred to as just '/'), or upload the entire folder (you can delete the `examples` folder first) to the host and configure your domain name to point to that folder.
```
Comment (paulswartz) [inline readme.md]: We don't have an `examples` folder so you don't need to mention it.

**Comment by paulswartz** [comment]
Thanks for doing this!  It's going to make it a lot easier for newcomers to get started with Finda.

**Comment by alexisylchan** [comment]
Changed img links to relative paths, removed mention of the examples folder, & removed changes to data.geojson

**Pull request merged**